diff --git a/src/App.jsx b/src/App.jsx
index 11e108f..3c47c76 100644
--- a/src/App.jsx
+++ b/src/App.jsx
@@ -7,9 +7,11 @@ import { Contact } from "@/sections/Contact";
 import { Footer } from "./layout/Footer";
 import { Achievements } from "./sections/Achievements";
 import { TechStack } from "./sections/TechStack";
+import { MatrixBackground } from "@/components/MatrixBackground";
 function App() {
   return (
     <div className="min-h-screen overflow-x-hidden">
+      <MatrixBackground />
       <Navbar />
       <main>
         <Hero />
